docs/hpc/applications/easybuild.md

```diff
@@ -38,195 +198,249 @@ archspec/0.1.4-GCCcore-11.3.0                            SCOTCH/7.0.1-iimpi-2022
+CFGS1=/sw-eb/software/EasyBuild/4.9.4/easybuild/easyconfigs
+[ ... ]
+ * $CFGS1/z/zlib/zlib-1.3.1-GCCcore-14.2.0.eb
+ * $CFGS1/z/zlib/zlib-1.3.1.eb
 ```
 
-Loading the development software stack on the login nodes and loading the GROMACS/2021.5-foss-2021b module.
+Your list will be much longer as all EasyConfig files which contain the string 'zlib' will be searched and displayed. We are only interested in the last two entries shown here. The `zlib-1.3.1-GCCcore-14.2.0.eb` would install `zlib` which is compiled using the `GCCcore-14.2.0` toolchain. That is not what we want. The `zlib-1.3.1.eb` does not have a toolchain version and thus automatically falls back to `SYSTEM`. 
+
+Installation is simple:
 
 ```console
-[username@login ~]$ module load tools/gpu-headnode
-[username@login ~]$ module load GROMACS/2021.3-foss-2021a-CUDA-11.3.1
-Loading GROMACS/2021.3-foss-2021a-CUDA-11.3.1
-  Loading requirement: GCCcore/10.3.0 zlib/1.2.11-GCCcore-10.3.0 binutils/2.36.1-GCCcore-10.3.0
-    GCC/10.3.0 numactl/2.0.14-GCCcore-10.3.0 XZ/5.2.5-GCCcore-10.3.0 libxml2/2.9.10-GCCcore-10.3.0
-    libpciaccess/0.16-GCCcore-10.3.0 hwloc/2.4.1-GCCcore-10.3.0 OpenSSL/1.1
-    libevent/2.1.12-GCCcore-10.3.0 UCX/1.10.0-GCCcore-10.3.0 libfabric/1.12.1-GCCcore-10.3.0
-    PMIx/3.2.3-GCCcore-10.3.0 OpenMPI/4.1.1-GCC-10.3.0 OpenBLAS/0.3.15-GCC-10.3.0
-    FlexiBLAS/3.0.4-GCC-10.3.0 gompi/2021a FFTW/3.3.9-gompi-2021a ScaLAPACK/2.1.0-gompi-2021a-fb
-    foss/2021a bzip2/1.0.8-GCCcore-10.3.0 ncurses/6.2-GCCcore-10.3.0 libreadline/8.1-GCCcore-10.3.0
-    Tcl/8.6.11-GCCcore-10.3.0 SQLite/3.35.4-GCCcore-10.3.0 GMP/6.2.1-GCCcore-10.3.0
-    libffi/3.3-GCCcore-10.3.0 Python/3.9.5-GCCcore-10.3.0 pybind11/2.6.2-GCCcore-10.3.0
-    SciPy-bundle/2021.05-foss-2021a networkx/2.5.1-foss-2021a CUDA/11.3.1 GDRCopy/2.2-GCCcore-10.3.0
-    UCX-CUDA/1.10.0-GCCcore-10.3.0-CUDA-11.3.1
+$ eb --trace zlib-1.3.1.eb
 ```
 
-Note that while you can view the modules from the production software stack on the login node, you cannot actually run them. For example, after loading GROMACS from the production software stack, you will not be able to run the GROMACS gmx application.
+The `--trace` will give you a bit more information. EasyBuild will first consult the installed module files to see if that particular version of `zlib` is already installed. If it is, it will not build it. If it is not, it will:
+
+- download the source package unless it can be found in the centrally managed source packages folder
+- compares the checksum of the downloaded file with the one stored in the EasyConfig file
+- unpack the source if required
+- installs the software as instructed in the EasyConfig file
+- tests the installation
+- writes the module file
+- finish off the installation by cleaning both the `buildpath` and `tmpdir` locations.
+
+In order to use the so installed software, you only need to load the `tools/eb-dev` module before loading the newly installed software package. 
```

Comment:
An additional line mentioning that new software can now be loaded as module will help readers on how to use the new software.

Reply:
I would have thought line 361 does imply that, does it not?

Reply:
The line 361 is telling that we need tools/eb-dev module before loading the newly installed software. How to load the newly installed software is missing. May be, I am missing something. I was suggesting something like "module load new_software".

Reply:
I think the trick here is the `before loading the newly installed software package` which implies to load the module. at least that is my reading. As discussed, we just leave it for now.

Reply:
I don't think that I agreed to this point. I am sorry if I sound like that. 

I would still strongly suggest to explicitly mention on how to use the new software. Since we have spent some time on it already, I am seeking third opinion. @davidhemp  Can you please suggest? Please feel free to choose and accept what seems suitable to you.

Reply:
It makes sense to have an example on how to use the newly installed software. Ideally an example job.

Reply:
Done